### DISCUSSION TITLE
Product images issue

### DISCUSSION THREAD

**Discussion by simulbeni** [PrestaShop/PrestaShop#37719] [Q&A]
### Prerequisites

- [x] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [x] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

I have a product with 210 combinations. Each combination is having 5 images. It's almost impossible to work with product edit page.

### Steps to reproduce

Create a product 

Add 210 combination
Upload 1000 images.
Assign them to respected combinations.

### PrestaShop version(s) where the bug happened

8.1.5

### PHP version(s) where the bug happened

8.2

**Comment by musicpanda** [comment]
Prestashop is optimized for the average user. For an eccentric shop like yours another tool may be better. You can try my free Prestools for example.
